argus/client/driver_matrix_tests/client.py

```diff
@@ -117,8 +117,11 @@ def get_test_cases(self, cases: list[ElementTree.Element]) -> list[RawMatrixTest
 
         return raw_cases
 
-    def get_driver_info(self, xml_name: str) -> dict[str, str]:
-        filename_re = r"(?P<name>[\w]*)\.(?P<driver_name>[\w]*)\.(?P<proto>v\d)\.(?P<version>[\d\.]*)\.xml"
+    def get_driver_info(self, xml_name: str, test_type: TestTypeType) -> dict[str, str]:
+        if test_type == "cpp":
+            filename_re = r"TEST-(?P<driver_name>[\w]*)-(?P<version>[\d\.]*)\.xml"
+        else:
+            filename_re = r"(?P<name>[\w]*)\.(?P<driver_name>[\w]*)\.(?P<proto>v\d)\.(?P<version>[\d\.]*)\.xml"
 
         match = re.match(filename_re, xml_name)
```

Comment:
Now function will have different return for cpp case. Is it expected?

Reply:
It will not affect, because used only <driver_name>

Reply:
It shouldn't, because used only <driver_name> which is defined in both returned results.